### DISCUSSION TITLE
Prevent destination_url to be duplicated on the short_urls table

### DISCUSSION THREAD

**Discussion by oussamaeljabbari** [ash-jc-allen/short-url#70] [Q&A] [answered]
how can I prevent duplication of destination url on the short_urls table ?
thanks for this useful package.

**Comment by ash-jc-allen** [comment, marked as answer]
Hey @ojpro I'm glad you like the package! 😄

Off the top of my head, I think the easiest way to prevent duplicate destination URLs, you could use something like this:

```php
if (ShortURL::where('destination_url', 'https://example.com')->exists()) {
    // There is already a short URL with the same destination here.
    // You could maybe do an early return or throw an exception?
}

$builder = new \AshAllenDesign\ShortURL\Classes\Builder();

$shortURLObject = $builder->destinationUrl('https://example.com')->make();
```

I've not run the code above so I'm not 100% sure if it'll work straight away, but it should hopefully get the point across. I hope this helps?

**Reply by prog-24** [reply]
#112
